include/internal/e_os.h

```diff
@@ -264,6 +264,8 @@ FILE *__iob_func();
 # else
 #  include <strings.h>
 # endif
+int ossl_strcasecmp(const char *s1, const char *s2);
+int ossl_strncasecmp(const char *s1, const char *s2, size_t n);
```

Comment:
include/crypto/cryptlib.h would be a better place for these declarations

Reply:
I agree.

Reply:
Moved to public header. Thanks!